src/m365/pp/commands/chatbot/chatbot-remove.spec.ts

```diff
@@ -0,0 +1,223 @@
+        });
+      }
+
+      throw new CommandError('Unknown case');
+    });
+
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://contoso-dev.api.crm4.dynamics.com/api/data/v9.1/bots(${validId})/Microsoft.Dynamics.CRM.PvaDeleteBot?tag=deprovisionbotondelete`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    sinonUtil.restore(Cli.prompt);
+    sinon.stub(Cli, 'prompt').callsFake(async () => (
+      { continue: true }
+    ));
+    await command.action(logger, {
+      options: {
+        debug: true,
+        environment: 'Default-eff8592e-e14a-4ae8-8771-d96d5c549e1c',
+        name: 'CLI 365 Chatbot'
+      }
+    });
+    assert(loggerLogToStderrSpy.called);
+  });
+
+  it('removes the specified chatbot without confirmation prompt', async () => {
+    sinon.stub(powerPlatform, 'getDynamicsInstanceApiUrl').callsFake(async () => envUrl);
+
+    sinon.stub(request, 'post').callsFake(async (opts) => {
+      if (opts.url === `https://contoso-dev.api.crm4.dynamics.com/api/data/v9.1/bots(${validId})/Microsoft.Dynamics.CRM.PvaDeleteBot?tag=deprovisionbotondelete`) {
+        return;
+      }
+
+      throw 'Invalid request';
+    });
+
+    await command.action(logger, {
+      options: {
+        debug: true,
+        environment: validEnvironment,
+        id: validId,
+        confirm: true
+      }
+    });
+    assert(loggerLogToStderrSpy.called);
```

Comment:
Right now we are just testing that a verbose message is logged. This is not really what we are looking for. Instead we should check if the postStub was called.